bundler/lib/bundler/man/gemfile.5.ronn

```diff
@@ -91,13 +91,13 @@ Each application _may_ specify a Ruby engine version. If an engine version is
 specified, an engine _must_ also be specified. If the engine is "ruby" the
 engine version specified _must_ match the Ruby version.
 
-    ruby "1.8.7", :engine => "jruby", :engine_version => "1.6.7"
+    ruby "1.8.7", engine:  "jruby", engine_version:  "1.6.7"
```

Comment:
It seems to be double spaces here after `:`, is that intended?

Reply:
good eyes, no it is not intended :(